src/tools/auth0/handlers/actions.js

```diff
@@ -153,6 +154,10 @@ export default class ActionHandler extends DefaultHandler {
         actionChanges.code = action.code;
       }
 
+      if (action.runtime !== found.runtime) {
```

Comment:
This is an existing bug: the `actionChanges()` function doesn't seem to be called at all and the actions handler always update and redeploy all existing actions (thus creating a new version every time deploy-cli runs import).

This also leads to where if one specifies the `runtime` property and then remove it from the configuration, it does not revert to the default runtime (because the update operation is using the PATCH endpoint).

I don't think this needs be fixed in this PR but would like to surface the issue. I'll file a Jira ticket.